src/Adapter/MSTest.TestAdapter/Helpers/DataSerializationHelper.cs

```diff
@@ -93,6 +105,10 @@ internal static class DataSerializationHelper
             }
 
             DataContractJsonSerializer serializer = GetSerializer(assemblyQualifiedName);
+#if NET7_0_OR_GREATER // TODO: Docs say it's available on net6.0, but that's not what I see here :/
```

Comment:
@nohwnd Do you see anything here please?

Reply:
![image](https://github.com/user-attachments/assets/9c489db9-5c2f-4d77-a4e9-63d64d15e650)

Reply:
@Evangelink Yes that's what I'm seeing, but docs say otherwose.